serverless/unsubscribe-digest/src/utils/cronitor.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { Cronitor } from '../interface'
+import config from '../config'
+import { Logger } from './logger'
+
+const logger = new Logger('db-backup')
+
+const CRONITOR_PING_URL = `https://cronitor.link`
```

Comment:
why did you choose to set two variables CRONITOR_PING_URL and CRONITOR_CODE instead of just one CRONITOR_URL? the two variables are always concatenated anyway

Reply:
Makes sense. Will use one variable.